multires-lyrics-search/README.md

```diff
@@ -1,109 +1,180 @@
 
-# Lyrics Live Search: Understanding the Concept of Chunk
-
-[![Jina](https://github.com/jina-ai/jina/blob/master/.github/badges/jina-badge.svg?raw=true  "We fully commit to open-source")](https://get.jina.ai)
+# Lyrics Live Search: Understanding the Concept of Chunks
 
 [![](demo.gif)](https://www.youtube.com/watch?v=GzufeV8AY_w)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**
 
-- [Use toy data](#use-toy-data)
-- [[Optional] Download full lyrics dataset](#optional-download-full-lyrics-dataset)
-- [Install](#install)
-- [Run](#run)
-- [View in Browser](#view-in-browser)
+- [🐍 Build the app with Python](#---build-the-app-with-python)
+- [Using the search interface and understanding your results](#using-the-search-interface-and-understanding-your-results)
+- [Overview of the files in this example](#overview-of-the-files-in-this-example)
+- [Flow diagram](#flow-diagram)
+- [Next steps, building your own app](#next-steps--building-your-own-app)
+- [🐳 Deploy the prebuild application using Docker](#---deploy-the-prebuild-application-using-docker)
+- [Community](#community)
 - [License](#license)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+This example shows you how to build a semantic search app powered by [Jina AI](http://www.jina.ai)'s neural search framework.  You can index and search song lyrics using state of art machine learning language models. This example helps teach you about the concept of chunking and it's importance in search systems. Once you've built the example, you can visual how the system is matching inputs to output via a custom website we have built. 
```

Comment:
inputs to outputs is a weird way to put it

Reply:
Any other suggestions?